modules/imap/functions.php

```diff
@@ -1412,6 +1412,56 @@ function snooze_dropdown($output, $unsnooze = false) {
     return $txt;
 }}
 
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('get_forward_content')) {
```

Comment:
Can you try to not use functions for this please?

Reply:
I just removed this function

Reply:
Please review